src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -1,15 +1,19 @@
-.bottomWrapper {
-  display: grid;
+.proteinSummary {
   margin-left: 178px;
   width: 695px;
 
   > div {
-    margin: 10px 0 10px 0;
+    padding: 10px 0 10px 0;
   }
 }
 
-.codingExonCount {
-  grid-column: 1/2;
+.proteinSummaryTopWrapper {
+  display: grid;
+  grid-template-columns: 300px 395px;
+}
+
+.downloadWrapper {
+  text-align: right;
```

Comment:
Will the `display: flex; justify-content: flex-end` communicate the intention better?

Reply:
This didn't work unfortunately so it is till using CSS grid and text alignment

Reply:
> This didn't work unfortunately

Works for me 🤷‍♂️ 

![image](https://user-images.githubusercontent.com/6834224/86664430-9a47c700-bfe6-11ea-9f45-c31bae1616b4.png)

Reply:
It still doesn't properly in my case:

![Screenshot 2020-07-07 at 12 03 52](https://user-images.githubusercontent.com/32512909/86771363-01e32e00-c04a-11ea-928d-5a073e894626.png)

The download component takes up the whole horizontal space as if it is a block element.